channels/mx.m3u

```diff
@@ -1,71 +1,75 @@
 #EXTM3U
 #EXTINF:-1 tvg-id="XHTITTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="" group-title="",A+ TV (XHTIT-TDT) (432p) [Offline]
 https://streams.the6tv.duckdns.org:2443/locals/SanDiego/xhtit-tdt2.m3u8
-#EXTINF:-1 tvg-id="XHTVMTDT.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (XHTVM-TDT) (480p) [Offline]
-https://aztkhlslive3-i.akamaihd.net/hls/live/777407/adn40mx/master.m3u8
+#EXTINF:-1 tvg-id="Adn40.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (480p)
```

Comment:
It is better to use the codes from the [iptv-org/epg](https://iptv-org.github.io/epg/index.html) where possible to be able to load the program later.

```suggestion
#EXTINF:-1 tvg-id="ADN40.mx" tvg-country="MX" tvg-language="Spanish" tvg-logo="https://i.imgur.com/x8lwsjM.png" group-title="",ADN 40 (480p)
```

Reply:
that strange ...  please  edit them  , it wont let me  ,github  tell me im not on the main branch